packit/cli/update.py

```diff
@@ -31,4 +31,8 @@ def update(config, dist_git_path, dist_git_branch, repo):
     """
     package_config = get_local_package_config(directory=repo.working_dir)
     api = PackitAPI(config=config, package_config=package_config)
-    api.sync_release(dist_git_branch, dist_git_path=dist_git_path)
+    try:
+        api.sync_release(dist_git_branch, dist_git_path=dist_git_path)
+    except PackitFedpkgException as pfe:
+        if config.debug:
+            raise
```

Comment:
okay, so now we just ignore the exception which is pretty bad, the error code should be >0

Reply:
Ok, now it returns `sys.exit(1)`

Reply:
I'm sorry, but that hasn't fixed the problem that I don't get ANY info (if debug is not on) in case of failure- all I have is exit code `1`, which is not very descriptive.